app/scripts/HiGlassComponent.js

```diff
@@ -4288,6 +4291,52 @@ class HiGlassComponent extends React.Component {
       return;
     }
 
+    const absX = nativeEvent.clientX;
+    const absY = nativeEvent.clientY;
+    const hoveredTiledPlot = this.getTiledPlotAtPosition(absX, absY);
+
+    // Find the tracks at the wheel position
+    if (this.numWheelCallbacks > 0) {
+      const relPos = clientPoint(this.topDiv, nativeEvent);
+      // We need to add the scrollTop
+      relPos[1] += this.scrollTop;
+      const hoveredTracks = hoveredTiledPlot
+        ? hoveredTiledPlot
+            .listTracksAtPosition(relPos[0], relPos[1], true)
+            .map(track => track.originalTrack || track)
+        : [];
+      const hoveredTrack = hoveredTracks.find(
+        track => !track.isAugmentationTrack
+      );
+
+      const relTrackPos = hoveredTrack
+        ? [
+            relPos[0] - hoveredTrack.position[0],
+            relPos[1] - hoveredTrack.position[1]
+          ]
+        : relPos;
+
+      const evtToPublish = {
+        x: relPos[0],
+        y: relPos[1],
+        relTrackX:
+          hoveredTrack && hoveredTrack.flipText
+            ? relTrackPos[1]
+            : relTrackPos[0],
+        relTrackY:
+          hoveredTrack && hoveredTrack.flipText
+            ? relTrackPos[0]
+            : relTrackPos[1],
+        track: hoveredTrack,
+        origEvt: nativeEvent,
+        sourceUid: this.uid,
+        hoveredTracks,
+        noHoveredTracks: hoveredTracks.length === 0
```

Comment:
Why is this necessary? Can't the use just do the length check themselves?

Reply:
It is not necessary, but it makes this event consistent with the existing mouse move event https://github.com/higlass/higlass/blob/develop/app/scripts/HiGlassComponent.js#L4044

Reply:
Interesting. OK, I'll hold my nose and let it slide :-)